That silly Debian patch changes the default dir to TRANSDIR.
This patch corrects that oversight so both work if we ever
use console-data, also by Debian.

--- kbd-1.12/src/loadunimap.c.orig	2006-08-24 13:48:11.000000000 +0100
+++ kbd-1.12/src/loadunimap.c	2006-08-24 13:49:07.000000000 +0100
@@ -28,7 +28,7 @@
 extern char *progname;
 extern int force;
 
-static char *unidirpath[] = { "", DATADIR "/" TRANSDIR "/", 0 };
-static char *unisuffixes[] = { "", ".sfm", ".uni", 0 };
+static char *unidirpath[] = { "", DATADIR "/" UNIMAPDIR "/", DATADIR "/" TRANSDIR "/", 0 };
+static char *unisuffixes[] = { "", ".uni", ".sfm", 0 };

 #ifdef MAIN
